<!DOCTYPE html>
<html lang="en">

	<head><script src="/livereload.js?mindelay=10&amp;v=2&amp;port=1313&amp;path=livereload" data-no-instant defer></script>
		<title> Experiences &middot; Yao LEI (Leo) GithubPage </title>

		<meta http-equiv="content-type" content="text/html; charset=utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="generator" content="Hugo 0.140.2">




<script src="https://code.jquery.com/jquery-3.1.1.min.js"   integrity="sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8="   crossorigin="anonymous"></script>


<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-+0n0xVW2eSR5OomGNYDnhzAbDsOXxcvSN1TPprVMTNDbiYZCxYbOOl7+AMvyTG2x" crossorigin="anonymous">
<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/js/bootstrap.bundle.min.js" integrity="sha384-gtEjrD/SeCtmISkJkNUaaKMoLD0//ElJ19smozuHV6z3Iehds+3Ulb9Bn9Plx0x4" crossorigin="anonymous"></script>


<link rel="stylesheet" href="https://use.fontawesome.com/releases/v6.7.1/css/all.css" integrity="sha384-QI8z31KmtR+tk1MYi0DfgxrjYgpTpLLol3bqZA/Q1Y8BvH+6k7/Huoj38gQOaCS7" crossorigin="anonymous">


<link rel="stylesheet" href="http://localhost:1313/css/nix.css">



<link rel="shortcut icon" href="/favicon/Favicon-32x32.png">



<link href="https://fonts.googleapis.com/css?family=Inconsolata%7COpen+Sans%7CConcert+One" rel="stylesheet">






	</head>

	<body>
		<header>
	<nav class="navbar navbar-dark bg-dark fixed-top navbar-expand-lg font-header">
		<div class="container-fluid">
			<a class="navbar-brand" id="green-terminal" href='http://localhost:1313/'>
				leiyao@GithubPage ~ $
			</a>
			<button type="button" class="navbar-toggler" data-bs-toggle="collapse" data-bs-target="#navbar-collapse-1" 
					aria-controls="navbar-collapse-1" aria-expanded="false">
				<span class="visually-hidden">Toggle navigation</span>
				<span class="navbar-toggler-icon"></span>
			</button>

			
			<div class="collapse navbar-collapse" id="navbar-collapse-1">
				<ul class="nav navbar-nav ms-auto">
					<li class="nav-item">
						<a class="nav-link" href='http://localhost:1313/'>
							/home/leiyao</a>
					</li>
					
					
					
					<li class="nav-item dropdown">
						
						<a class="nav-link" href="http://localhost:1313/experience/">~/experience</a>
						
					</li>
					
					
					<li class="nav-item dropdown">
						
						<a class="nav-link" href="http://localhost:1313/publication/">~/publication</a>
						
					</li>
					
					
					<li class="nav-item dropdown">
						
						<a class="nav-link" href="http://localhost:1313/contacts/">~/contacts</a>
						
					</li>
					
				</ul>
			</div>
		</div>
	</nav>
</header>

		<div class="flex-wrapper">
			<div class="container wrapper">
				<div class="row">
					<div class="col-xs-12 text-center">
						<h1 id=>Experiences</h1>
					</div>
				</div>
				<ul id="post-list">
	
	<li>
		<div class="post-list-item">
			<div class="post-header">
				<h4 class="post-link"><a href="http://localhost:1313/experience/4/">Distinct genetic risk loci between biopsy-proven renal and non-renal lupus: a 10-year longitudinal cohort. (Oxford Rheumatology)</a></h4>
				
			</div>
			<div class="post-summary"><p><p>Systemic lupus erythematosus (SLE) is a prototypical autoimmune disease that can affect multiple organ systems. Lupus nephritis (LN), characterized by inflammation of the kidneys, is a severe subtype of SLE. In this study, using 10 years of clinical follow-up data, we classified patients into LN and non-nephritic (NLN) groups. First, we replicated the finding that the LN group has a higher distribution of polygenic risk scores compared to the NLN group. We also identified that loci in HLA region, TNFAIP3, and BLK are significantly associated with the LN group but not in NLN group. Furthermore, using heterogeneity testing, we found that loci in ELF1, OX40, DUSP22, and TPCN2 differ significantly between the LN and NLN groups.</p></p></div>
			<div class="post-list-footer text-center">
				
			</div>
		</div>
	</li>
	
	<li>
		<div class="post-list-item">
			<div class="post-header">
				<h4 class="post-link"><a href="http://localhost:1313/experience/5/">Identification of common and rare variants that associated with growth hormone deficiency using whole-genome sequencing.</a></h4>
				
			</div>
			<div class="post-summary"><p><p>Growth hormone deficiency (GHD) is a subtype of human short stature caused by insufficient growth hormone production. Although human height is a complex trait, GHD is often considered a Mendelian disease. In this study, using the our in-house WGS as control, we plan to collect whole-genome sequencing (WGS) data from 1,200 GHD patients and perform genome-wide association studies (GWAS) and rare variant association studies (RVAS) to identify common and rare variants or genes associated with GHD.</p></p></div>
			<div class="post-list-footer text-center">
				
			</div>
		</div>
	</li>
	
	<li>
		<div class="post-list-item">
			<div class="post-header">
				<h4 class="post-link"><a href="http://localhost:1313/experience/6/">Identification of common and rare variants that associated with systemic lupus erythematosus using whole-genome sequencing.</a></h4>
				
			</div>
			<div class="post-summary"><p><p>Systemic lupus erythematosus (SLE) is a complex autoimmune disease. Numerous genome-wide association studies (GWAS) have been conducted, and more than 300 common loci have been reported to be associated with SLE. However, these loci explain only a limited proportion of the disease&rsquo;s heritability. The goal of this study is to uncover the missing heritability of SLE. In collaboration with the Hong Kong Genome Institute, West China Hospital of Sichuan University, and Jining Medical University, we collected whole-genome sequencing (WGS) data from 2,000 SLE patients and 40,000 control individuals. Through this study, we aim to identify common and rare genetic variants and loci for SLE.</p></p></div>
			<div class="post-list-footer text-center">
				
			</div>
		</div>
	</li>
	
	<li>
		<div class="post-list-item">
			<div class="post-header">
				<h4 class="post-link"><a href="http://localhost:1313/experience/3/">Association of genetic variation on X chromosome with systemic lupus erythematosus in both Thai and Chinese populations. (BMJ lupus science &amp; Medicine)</a></h4>
				
			</div>
			<div class="post-summary"><p><p>Genome-wide association study (GWAS) is a well-established method for identifying loci associated with human complex diseases. Several GWAS have been conducted on the X chromosome to discover candidate loci for systemic lupus erythematosus (SLE). However, studies in the Thai population are still lacking. In this study, we collected SNP array data from Thai individuals and performed X chromosome-specific GWAS, combining these data with our in-house Hong Kong cohort. We successfully replicated previously reported loci, including TEME187-IRAK-MECP2, PRPS2, TLR7, and GPR173, and identified two novel candidate loci associated with SLE: LOC389895-SOX3 and CT83-KLHL13. Additionally, we confirmed a higher frequency of trisomy X among SLE patients compared to controls in both the Thai and Hong Kong populations.</p></p></div>
			<div class="post-list-footer text-center">
				
			</div>
		</div>
	</li>
	
	<li>
		<div class="post-list-item">
			<div class="post-header">
				<h4 class="post-link"><a href="http://localhost:1313/experience/2/">Phenotype-based prioritization enhances diagnosis rate of patients with Titinopathy. (Manuscript preparing)</a></h4>
				
			</div>
			<div class="post-summary"><p><p>Titinopathy is a Mendelian muscle disease caused by mutations in the TTN gene. However, due to the clinical and genetic heterogeneity of muscle diseases, mutations in TTN are often overlooked. To improve the diagnostic rate, we collected patients who had been clinically diagnosed with muscle diseases but in whom no confirmed causal mutations had been previously identified, and performed whole-genome sequencing (WGS). Using the latest phenotype-based prioritization strategies, we identified candidate causal variants in the TTN gene for approximately 40% of these patients.</p></p></div>
			<div class="post-list-footer text-center">
				
			</div>
		</div>
	</li>
	
	<li>
		<div class="post-list-item">
			<div class="post-header">
				<h4 class="post-link"><a href="http://localhost:1313/experience/1/">Genome-wide association study on Northern Chinese identifies KLF2, DOT1L and STAB2 associated with systemic lupus erythematosus. (Oxford Rheumatology)</a></h4>
				
			</div>
			<div class="post-summary"><p><p>Systemic lupus erythematosus is a complex autoimmune disease. We collected SNP array data form 522 SLE and 1017 controls samples from Jining, Shandong Province, China. Genome-wide association study was performed on these samples, followed-by a meta-analysis that integrated our in-house Chinese and European SLE cohorts. Thought this analysis, we identified KLF2, DOT1L, and STAB2 as novel SLE associated loci.</p></p></div>
			<div class="post-list-footer text-center">
				
			</div>
		</div>
	</li>
	
</ul>

				



			</div>
			<footer class="footer text-center">
	<p>
		Copyright &copy; 2025 Yao LEI (Leo)
	</p>
</footer>

		</div>
	</body>
</html>
